node/src/lib/logger.js

```diff
@@ -0,0 +1,44 @@
+const { createLogger, format, transports } = require('winston');
+const _ = require('lodash');
+const { argumentsMap } = require('../config/arguments');
+
+let secrets = [];
+
+const getSecureSecret = secret => {
```

Comment:
this is a little problematic 
```
> getSecureSecret('abc');
'abc'
> getSecureSecret('12345');
SyntaxError: Invalid regular expression: /^.{1,0}/: numbers out of order in {} quantifier
> getSecureSecret('123');
'123'
> getSecureSecret('123aa');
SyntaxError: Invalid regular expression: /^.{1,0}/: numbers out of order in {} quantifier
```

Reply:
you can just use a static `'******'` here

Reply:
25cf208